docs/topics/spiders.rst

```diff
@@ -59,16 +59,22 @@ scrapy.Spider
 
    .. attribute:: name
 
-       A string which defines the name for this spider. The spider name is how
-       the spider is located (and instantiated) by Scrapy, so it must be
-       unique. However, nothing prevents you from instantiating more than one
-       instance of the same spider. This is the most important spider attribute
-       and it's required.
+        A string which defines the name for this spider.
 
-       If the spider scrapes a single domain, a common practice is to name the
-       spider after the domain, with or without the `TLD`_. So, for example, a
-       spider that crawls ``mywebsite.com`` would often be called
-       ``mywebsite``.
+        If :setting:`SPIDER_LOADER_REQUIRE_NAME` is ``True`` and you use the
+        default Scrapy spider loader (see :setting:`SPIDER_LOADER_CLASS`), a
+        non-empty name is required for the spider to be discoverable by the
+        Scrapy commands :command:`crawl`, :command:`list`, and
+        :command:`runspider`.
+
+        The spider name must be unique to one spider class. If two or more
+        spiders have the same name, Scrapy commands :command:`crawl` and
+        :command:`runspider` will only be able to run one of the spiders.
+
+        If the spider scrapes a single domain, a common practice is to name the
+        spider after the domain, with or without the `TLD`_. So, for example, a
+        spider that crawls ``mywebsite.com`` would often be called
+        ``mywebsite``.
```

Comment:
I think this has an extra indent of 1?

Reply:
It was kind of on purpose, because the original uses 7 spaces (the first indentation uses 3, the second 4), but since the issue affects the whole class definition I’ll revert.